context/app/static/js/pages/Workspaces/Workspaces.jsx

```diff
@@ -1,25 +1,46 @@
 import React from 'react';
 
+import EmailIconLink from 'js/shared-styles/Links/iconLinks/EmailIconLink';
 import { useAppContext } from 'js/components/Contexts';
 import WorkspacesTitle from 'js/components/workspaces/WorkspacesTitle';
 import { LightBlueLink } from 'js/shared-styles/Links';
 import WorkspacesAuthenticated from 'js/components/workspaces/WorkspacesAuthenticated';
-
 import { StyledDescription } from './style';
 
 function Workspaces() {
   const { isAuthenticated, isWorkspacesUser } = useAppContext();
+
+  if (!isAuthenticated) {
+    return (
+      <>
+        <WorkspacesTitle />
+        <StyledDescription>
+          The workspaces feature is only available if logged in and is part of the allowed Globus group.{' '}
```

Comment:
```suggestion
          The workspaces feature is only available if logged in and is part of the allowed Globus group.{' '}
```
@tsliaw I don't typically get into the text, but I'm not sure if this is quite right.

Reply:
Brainstorming some ideas:
`only available if _user_ is logged in and is part of the allowed Globus group` -> If using the third person for instructions
`only available if you are logged in and are part of the allowed Globus group` -> if using the second person for instructions
`only available when logged in and part of the allowed Globus group` -> avoids both first and second person

Reply:
Since the other messages are using second person, I will keep it consistent and use the second person as well. Thanks for that suggestion Nick!